src/TrainingApi.http

```diff
@@ -0,0 +1,8 @@
+@HostAddress = http://localhost:5198
```

Comment:
Nit: Maybe we want to move these to a top-level `http` directory or something so they are easier to discover?

Reply:
I put the file here, and gave it this specific name, because I think that's how it would appear in a project created with "dotnet new webapi". But happy to move it up if you think that is best.